### PR TITLE
DM-23570: Switch document look and feel to Rubin Observatory

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: texmf/ls-R [hunk 1]
```diff
@@ -396,6 +396,7 @@ LSSToptions.tex
 lsst
 
 ./tex/latex/lsst:
+.lsstdoc.cls.swp
```
Comment (timj) [inline texmf/ls-R:399]: I don't think this should be here.

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 2]
```diff
@@ -474,24 +455,24 @@
...
+   The control is on the most recent digital document with this Handle in the Rubin digital archive and not printed versions.
    }
    \def\@controlFootText{%
    The contents of this document are subject to configuration control and may not be changed, altered, or their provisions waived without prior approval.
    }
    \fi
    \if@dmccb
    \def\@controlText{%
-   This LSST document has been approved as a Content-Controlled Document by the LSST DM Change Control Board.
+   This Rubin document has been approved as a Content-Controlled Document by the Rubin DM Change Control Board.
```
Comment (timj) [inline texmf/tex/latex/lsst/lsstdoc.cls:469]: I think we probably want to be saying "Rubin Observatory" here whenever "Rubin" is used.

File: texmf/tex/latex/lsst/lsstdoc.cls [hunk 3]
```diff
@@ -722,7 +703,7 @@
 \setlength{\lsstheadlogooffset}{\headextrawidth}
 
 % Text to use below the logo in the header
-\newcommand{\headertext}{\tiny LARGE SYNOPTIC SURVEY TELESCOPE}
+\newcommand{\headertext}{\tiny Vera C. Rubin Observatory}
```
Comment (timj) [inline texmf/tex/latex/lsst/lsstdoc.cls:706]: You might want to do a quick test with pdflatex to see if that hard coded length below needs to be tweaked.

File: texmf/ls-R (hunk 1 above)
Comment (womullan) [reply texmf/ls-R:399]: should not be

File: texmf/tex/latex/lsst/lsstdoc.cls (hunk 2 above)
Comment (womullan) [reply texmf/tex/latex/lsst/lsstdoc.cls]: ok

File: texmf/tex/latex/lsst/lsstdoc.cls (hunk 3 above)
Comment (womullan) [reply texmf/tex/latex/lsst/lsstdoc.cls:706]: currently not used since the logo is "Vera C Rubin Observatory" - when we get a log it may come back ..

**Pull request merged**